crates/orchestrator/src/main.rs

```diff
@@ -1,14 +1,29 @@
 use dotenvy::dotenv;
-use orchestrator::config::init_config;
+use opentelemetry::global;
 use orchestrator::queue::init_consumers;
 use orchestrator::routes::app_router;
+use orchestrator::telemetry;
+use orchestrator::{config::init_config, telemetry::OTEL_COLLECTOR_ENDPOINT};
+use tracing::Level;
+use tracing_opentelemetry::OpenTelemetryLayer;
+use tracing_subscriber::layer::SubscriberExt as _;
+use tracing_subscriber::util::SubscriberInitExt as _;
 use utils::env_utils::get_env_var_or_default;
 
 /// Start the server
 #[tokio::main]
 async fn main() {
     dotenv().ok();
-    tracing_subscriber::fmt().with_max_level(tracing::Level::INFO).with_target(false).init();
+
+    if !OTEL_COLLECTOR_ENDPOINT.is_empty() {
+        telemetry::init();
+        let tracer = telemetry::global_tracer().clone();
+        tracing_subscriber::registry()
+            .with(tracing_subscriber::filter::LevelFilter::from_level(Level::INFO))
```

Comment:
can this level be taken from the env?

Reply:
Alright, Implemented [here](https://github.com/madara-alliance/madara-orchestrator/pull/131/commits/ff2db549fab515b0dac23140e0a00a770eb24c04).